db/psychologists.ts

```diff
@@ -307,6 +307,20 @@ const reactivate = async (): Promise<number> => db(psychologistsTable)
     .andWhere('inactiveUntil', '<=', (new Date()).toISOString())
     .update({ active: true, inactiveUntil: null });
 
+const inactive = async (
+  token: string,
+  reason: string,
+): Promise<void> => {
+  const psy = await db('inactive_token').where({ token }).first();
```

Comment:
Shouldn't we filter on `confirm : false` too ?

Reply:
Why ?